components/camera/camera.go

```diff
@@ -249,12 +342,12 @@ func GetImagesFromGetImage(
 		logger.Warnf("requested mime type %s, but received %s", mimeType, resMimetype)
 	}
 
-	img, err := rimage.DecodeImage(ctx, resBytes, utils.WithLazyMIMEType(resMetadata.MimeType))
```

Comment:
add comment to func docstring that this method is deprecated and why it was needed